OK, I'm a bit mystified by why this is failing this time round:

```
CMake Error at CMakeLists.txt:686 (message):
  The target `RISCV' is experimental and must be passed via
  LLVM_EXPERIMENTAL_TARGETS_TO_BUILD.
```

... when I'm passing `RISCV` by exactly that variable.  Anyone have any ideas?

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -95,7 +95,8 @@ parts:
     - -DCMAKE_BUILD_TYPE=Release
     - -DCOMPILER_RT_INCLUDE_TESTS=OFF
     - -DLLVM_BINUTILS_INCDIR=/usr/include
-    - -DLLVM_TARGETS_TO_BUILD=X86\;AArch64\;ARM\;PowerPC\;NVPTX
+    - -DLLVM_TARGETS_TO_BUILD=AArch64;ARM;Mips;MSP430;NVPTX;PowerPC;X86
+    - -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=RISCV;WebAssembly
     stage:
     - -*
     prime:

